n2y/blocks.py

```diff
@@ -868,9 +868,9 @@ def _get_synced_block_children(self):
 
     def to_pandoc(self):
         if not self.shared:
-            self.client.logger.warning(
```

Comment:
Hey @sHermanGriffiths is this the fans warning that we keep seeing printed? If so, I don't think we need comment out the warning. Instead, we should delete it and write a comment explaining why we can't reliably print a warning since there are a lot of false positives etc. etc.